komand/server.py

```diff
@@ -177,6 +177,91 @@ def plugin_info():
             }
             return jsonify(PluginInfoSchema().dump(response))
 
+        @app.route("/actions")
+        def actions():
+            """Plugin actions list endpoint.
+            ---
+            get:
+              summary: Get list of plugin actions
+              description: Get InsightConnect plugin all actions
+              responses:
+                200:
+                  description: InsightConnect Plugin actions list to be returned
+                  schema:
+                    type: array
```

Comment:
```suggestion
                    type: list
```
Nitpick, but Python uses lists